app/controllers/VatDetailsController.scala

```diff
@@ -57,7 +57,8 @@ class VatDetailsController @Inject()(val messagesApi: MessagesApi,
         customerInfo <- accountDetailsCall
         nextReturn <- returnObligationsCall
         nextPayment <- if (retrieveHybridStatus(customerInfo)) Future.successful(Right(None)) else paymentObligationsCall
-        mandationStatus <- mandationStatusService.getMandationStatus(user.vrn)
+        mandationStatus <- if(appConfig.features.submitReturnFeatures()) {
+          mandationStatusService.getMandationStatus(user.vrn) } else { Future.successful(Right(MandationStatus("Disabled"))) }
```

Comment:
This else hasn't been covered within the tests. Can a quick test be added to the Controller Spec just cover this please?